salt/states/cmd.py

```diff
@@ -1048,7 +1048,7 @@ def script(
         return ret
 
     if runas and salt.utils.platform.is_windows() and not password:
-        ret["commnd"] = "Must supply a password if runas argument is used on Windows."
+        ret["command"] = "Must supply a password if runas argument is used on Windows."
```

Comment:
I would suspect this is supposed to be `comment` if it's part of the state return. I don't know what the state does with additional keys... they may be ignored